src/appComponents/twitterwall/singletweet.js

```diff
@@ -14,22 +14,22 @@ function tweet($timeout,$rootScope) {
         array:"="
     },
     template: 
-            '<div class="container-tweet">'+
-            '<div class="animate" ng-repeat="photo in array">'+
-            '<div ng-if="photo.id_str===selectedid"  class="tweet">'+
-                '<img ng-class="{\'animation\':photo.id_str===selectedid}"  class="image" on-error-src="https://pbs.twimg.com/profile_images/1141238022/fossasia-cubelogo.jpg" ng-src=\'{{photo.user.profile_image_url_https}}\'>'+
-            '</div>'+
-            '<div ng-if="photo.id_str===selectedid" class="heading-tweet">'+
-                '{{photo.screen_name}}'+
-            '</div>'+
-            '<div ng-if="photo.id_str===selectedid" class="text">'+
-                '{{photo.text}}'+
-            '<div class="mention" ng-if="photo.id_str===selectedid" ng-repeat="mention in photo.mentions" class="info">'+
-                '@{{mention}}'+
-            '</div>'+
-            '</div>'+
-            '</div>'+
-            '</div>',
+            "<div class=\"container-tweet\">"+
+            "<div class=\"animate\" ng-repeat=\"photo in array\">"+
+            "<div ng-if=\"photo.id_str===selectedid\"  class=\"tweet\">"+
+                "<img ng-class=\"{\'animation\':photo.id_str===selectedid}\"  class=\"image\" on-error-src=\"https://pbs.twimg.com/profile_images/1141238022/fossasia-cubelogo.jpg\" ng-src=\"{{photo.user.profile_image_url_https}}\">"+
+            "</div>"+
+            "<div ng-if=\"photo.id_str===selectedid\" class=\"heading-tweet\">"+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Infix operators must be spaced.](https://www.codacy.com/app/triplez/open-event-webapp/file/2443856941/issues/source?bid=3233857&fileBranchId=3245777#l22)